src/Components/LogExplorationPage.tsx

```diff
@@ -36,3 +36,5 @@ export function LogExplorationView() {
 
   return <scene.Component model={scene} />;
 }
+
+export default LogExplorationView;
```

Comment:
Are default exports a requirement for code splitting?

Reply:
I don't think it's required but it makes the imports a bit easier to write